README.md

```diff
@@ -27,9 +21,41 @@ Once you are logged in, click on the "Personal" text on the top right of your br
 1. Click on "+ Create new secret key" and give it a name.
 1. Copy the key and paste it in the .env file in the root directory of this project. Replacing "your key here" 
 
+## Setting up the runtime environment
+Copy the .env.example into a .env in the root of this project directory and populate your OPENAI_API_KEY:
+ ```shell
+ OPENAI_API_KEY=your key here
+ ```
+
+*optional: if you want to run this on a port other than the default gradio port of 7860 you  can add 
+GRADIO_PORT=<your port here> to the .env file as well*
+
+### Cloud Based Conversation Memory
+Follow the link and the instructions in the README to set up the [Griptape Structure Chatbot](https://github.com/griptape-ai/griptape-structure-chatbot) structure and CDK. 
+This will create your lambda endpoint and [Griptape Structure ID](https://cloud.griptape.ai/structures).
+```shell
+GT_STRUCTURE_ID=<your-structure-id>
+LAMBDA_ENPOINT=<your-lambda-endpoint>
+```
 
 ## Running the Demo
 
+If you would like to run the demo locally, then do NOT define these variables in your .env. 
```

Comment:
Instead of "not define" should we say "comment out"?

Reply:
Yes, that's likely easier